CHANGELOG.md

```diff
@@ -3,6 +3,14 @@
 ## [2.3.2](https://launchpad.net/mixxx/+milestone/2.3.2) (Unreleased)
 
 * Improve robustness of file type detection by considering the actual MIME type of the content. [lp:1445885](https://bugs.launchpad.net/mixxx/+bug/1445885) [#4356](https://github.com/mixxxdj/mixxx/pull/4356) [#4357](https://github.com/mixxxdj/mixxx/pull/4357)
+* Fix/Improve Appstream metainfo
+  [#4344](https://github.com/mixxxdj/mixxx/pull/4344)
+  [#4346](https://github.com/mixxxdj/mixxx/pull/4346)
+  [#4349](https://github.com/mixxxdj/mixxx/pull/4349)
+* Playlist: enable sorting by color [#4352](https://github.com/mixxxdj/mixxx/pull/4352) [lp:1945976](https://bugs.launchpad.net/mixxx/+bug/1945976)
+* CMake: Adjust the download directory and name of external dependencies [#4511](https://github.com/mixxxdj/mixxx/pull/4511)
```

Comment:
@uklotzde is this relevant to package maintainers where build servers mustn't access the internet?

If so, this should go into a packaging section. If not, this doesn't need to be mentioned at all.